### ISSUE TITLE
If Brave is not set as default upon first launch, `brave://settings/getStarted` page should show `Make default` button

### ISSUE THREAD

**Issue by MadhaviSeelam** [brave/brave-browser#27554]
## Description 

When Brave is not set as default in `brave://welcome` first screen and complete (with or without import) onboarding process, and navigate to `brave://settings/getStarted` page, `Brave is your default browser` message shown. It should show `Make default` button instead. Opened a link from Slack to confirm, it opened to Brave Beta.  

## Steps to Reproduce

1. new profile
2. launch Brave
3. click `Skip` on `brave://welcome` screen
5. click `Skip` on `Import Settings` screen
6. click finish
7. new-tab page is shown
8. go to `brave://settings/getStarted`

## Actual result:

`Brave://settings/getStarted` page shows `Brave is your default browser` message
<img width="547" alt="image" src="https://user-images.githubusercontent.com/98358127/209837862-4bb30c29-786f-4b4e-842d-bad5af4809ad.png">

## Expected result:
`Brave://settings/getStarted` page should show `Make default` button 
<img width="647" alt="image" src="https://user-images.githubusercontent.com/98358127/209838531-8be76f8b-d8b5-42a5-a767-369d031f349e.png">

## Reproduces how often: 

Easily

## Brave version (brave://version info)

```
Brave | 1.47.149 Chromium: 109.0.5414.46 (Official Build) beta (64-bit)
-- | --
Revision | 6e36b77363ef3febbe792af680fa1367993ddcf0-refs/branch-heads/5414@{#709}
OS | Windows 11 Version 22H2 (Build 22621.963)
```
## Version/Channel Information:

- Can you reproduce this issue with the current release? N/A
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

@spylogsster @rebron

**Comment by rebron** [comment]
cc: @nullhook

**Comment by rebron** [comment]
Closing as invalid. Discussed with @MadhaviSeelam

**Comment by stephendonner** [comment]
> Closing as invalid. Discussed with @MadhaviSeelam

@rebron @MadhaviSeelam As this might affect me on macOS for verification, I'd love to know the reasoning, as well.  Thanks!

**Comment by MadhaviSeelam** [comment]
@stephendonner - So the reason being I've already set Brave Beta (for some testing) as a default in the windows settings.  These settings are over riding `Skip` in the UI.  

So here are the two scenarios I tested: 
Scenario 1: 
Pre-requisite: Make sure `Microsoft Edge` is default browser Or do a Reset in `Reset all default all apps`  (Windows Settings>>Apps>>Default Apps>
- Follow the STR from the description
- `Make Default` button shown as expected 

Scenario 2: 
Continue from scenario 1:
      - delete the profile
      - switched default file types (html, .htm, .svg, http, https) to Beta in Windows Settings>>Apps>>Default Apps>

- Followed STR from the description and this time, I didn't get `Make default` button.  
- Since this is an edge case or not real use case, decided to close. 

![image](https://user-images.githubusercontent.com/98358127/209882768-fdab3439-2082-4217-93d8-eae5c9bd2bca.png)
